### DISCUSSION TITLE
Browser Support Documentation?

### DISCUSSION THREAD

**Discussion by mwisner** [chakra-ui/chakra-ui#5315] [Q&A]
Hi,

Is there any "official" documentation around browser support provided by `chakra-ui`? 

It seems like Safari 13 support was removed as a result of this patch change: https://github.com/chakra-ui/chakra-ui/commit/53e2df4f9cbe7fc2fad69d1ee49a1e788811467a

https://github.com/chakra-ui/chakra-ui/issues/5197.

I'm not sure if Safari 13 was ever officially supported, but it seemed to have worked fine for us up until now (around 1.7). Given the "won't fix" response in https://github.com/chakra-ui/chakra-ui/issues/5197. Are there any Chakra UI browser support docs that should be updated to reflect no Safari 13 support?

**Comment by noobinthisgame** [comment]
Hey @mwisner,

it is indeed planned to have an FAQ entry about browser support. See [this issue](https://github.com/chakra-ui/chakra-ui-docs/issues/117)
Feel free to take on this issue if you want :)
